editor/project_manager.cpp

```diff
@@ -2866,8 +2847,9 @@ ProjectManager::ProjectManager() {
 	vb->add_child(center_box);
 
 	tabs = memnew(TabContainer);
-	center_box->add_child(tabs);
 	tabs->set_anchors_and_offsets_preset(Control::PRESET_FULL_RECT);
+	tabs->set_current_tab(0);
```

Comment:
What's this line for?

Reply:
I don't know, it was a part of the old `_load_recent_projects` method, but it didn't make sense to me there either. The only context where I thought it might be applicable is the initialization, so I moved it here. But I can remove it.